qemu/tests/cfg/boot_from_virtiofs.cfg

```
@@ -28,4 +28,6 @@
     backend_mem_mem0 = memory-backend-file
     share_mem = yes
     kernel_params = "root=virtiofs:${fs_target} console=ttyS0"
+    aarch64:
+        kernel_params = "root=virtiofs:${fs_target} console=ttyAMA0"
```

We should consider all architectures, in my opinion:
```suggestion
    x86_64, i386:
        console_dev = "ttyS0,115200"
    ppc64le, ppc64:
        console_dev = "hvc0,38400"
    aarch64:
        console_dev = "ttyAMA0,38400"
    s390x:
        console_dev = "ttysclp0,38400"
    kernel_params = "root=virtiofs:${fs_target} console=${console_dev}"
        
```